pkg/operator/operator.go

```diff
@@ -67,10 +71,20 @@ import (
 	"github.com/aws/karpenter/pkg/providers/version"
 )
 
+var BuildInfo = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: metrics.Namespace,
+		Name:      "build_info",
+		Help:      "A metric with a constant '1' value labeled by version from which karpenter was built.",
+	},
+	[]string{"version", "golang_version"})
+
 func init() {
 	lo.Must0(apis.AddToScheme(scheme.Scheme))
 	v1alpha5.NormalizedLabels = lo.Assign(v1alpha5.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": corev1.LabelTopologyZone})
 	corev1beta1.NormalizedLabels = lo.Assign(corev1beta1.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": corev1.LabelTopologyZone})
+	crmetrics.Registry.MustRegister(BuildInfo)
+	BuildInfo.WithLabelValues(operator.Version, runtime.Version()).Set(1)
```

Comment:
You could consider using `changeset.Get()` to grab the SHA that is injected into the logger here into the metric: https://github.com/kubernetes-sigs/karpenter/blob/main/pkg/operator/logging/logging.go#L91

Reply:
Thank you for review and suggestion 👍 

I just yesterday noticed `sha` somewhere in karpenter logs, missed it initially when checked code.

Reply:
Done, run tests in test EKS to check how this new metric works, all looks good to me.